maven-project-crawler/src/main/java/com/exasol/projectkeeper/MavenProjectCrawlerMojo.java

```diff
@@ -39,6 +39,11 @@ public class MavenProjectCrawlerMojo extends AbstractMojo {
     // [impl -> dsn~eclipse-prefs-java-version~1]
     @Override
     public void execute() {
+        if (projectsToCrawl == null || projectsToCrawl.isBlank()) {
```

Comment:
```suggestion
        if (this.projectsToCrawl == null || this.projectsToCrawl.isBlank()) {
```

Reply:
In case Mojos have a different style-guide recommending or even requiring to skip the prefix `this.`, please ignore my suggestion.